packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -68,8 +71,10 @@ public function __construct(
 
     /**
      * @Route("/dashboard/")
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function dashboardAction()
+    public function dashboardAction(Request $request)
```

Comment:
This would be considered a BC break. Please, extract the redirect into a new controller action to avoid it.

Reply:
used action within formCreate method